src/init.rs

```diff
@@ -127,15 +131,12 @@ impl Init {
     }
 
     fn init_build(&self, family: &Family) -> Result<(), Error> {
-        match family {
-            Family::STM32 => self.create_file(
-                "build.rs",
-                include_str!("templates/build.rs.stm32.template"),
-            ),
-            Family::NRF(_) => {
-                self.create_file("build.rs", include_str!("templates/build.rs.nrf.template"))
-            }
-        }
+        let tpl = match family {
+            Family::STM32 => include_str!("templates/build.rs.stm32.template"),
+            Family::NRF(_) => include_str!("templates/build.rs.nrf.template"),
+        };
+
+        self.create_file("build.rs", tpl)
```

Comment:
Maybe since you are on a "readability crusade" `tpl` should be renamed to `template`.

Reply:
Done